app/views/home/companies.html.erb

```diff
@@ -1,9 +1,114 @@
-<h3>How to make more money through OSS Fridays</h3>
-<p class="lead">
-Open source software is the infrastructure that the entire technology sector is built on. Infrastructure is often taken for granted and under-resourced but is essential to running businesses effectively. Take control rather than relying entirely on others to maintain the software your business depends on. Small, regular contributions to open source allow you to stop maintaining internal versions of open source software in favour of the projects maintaining your patches, help your employees learn different ways of working that they can bring back into making your organisation more efficient and build the reputation of your organisation aiding community outreach efforts and hiring.
-</p>
-
-<h3>How to start OSS Fridays in your organisation</h3>
-<p class="lead">
-Encourage your managers, coworkers and reports to read the <a href="/contributors">Contributors page</a> and <a href="https://opensource.guide/how-to-contribute/">How to Contribute to Open Source</a> on Open Source Guides if they haven't already contributed to open source. If they have but don't currently do so every Friday, provide support, resources and time for them do so. If they already maintain open source projects, encourage them to read the <a href="/maintainers">Maintainers page</a> and help others in your organisation contribute to their project.
-</p>
+<div class="bg-gray mb-6 border-bottom jumbotron px-3">
+  <div class="container-md">
+    <h2 class="alt-h1 text-purple text-center">Your business depends on it</h2>
```

Comment:
This was actually intended.  The thinking being that this should semantically be an `<h2>` because the `<h1>` is the site title and this is the page title, but visually it should be the biggest thing on the page. FWIW, the site title is `<h1 class="alt-h4">`. I know it's weird, but I've seen this pattern a lot in other uses of Primer.